bigquery_etl/cli/query.py

```diff
@@ -1556,15 +1556,17 @@ def deploy(
         sys.exit(1)
     client = bigquery.Client()
 
-    query_files = paths_matching_name_pattern(name, sql_dir, project_id)
+    query_files = paths_matching_name_pattern(name, sql_dir, project_id, ["query.*"])
```

Comment:
We do allow views having schema files, but we do not want to publish these views as tables, so filter them out here